Package.resolved

```diff
@@ -140,8 +140,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/torusresearch/tss-client-swift.git",
       "state" : {
-        "revision" : "40246d5e3ff1d2c97d41846576f7a81d58858981",
-        "version" : "4.0.0"
+        "revision" : "3beedc78716d9681aaa1dbe110cdfd33090a8fff",
+        "version" : "4.0.2"
```

Comment:
Revert this to 4.0.0, tag 4.0.2 was sitting on a separate unmerged branch that will be discarded. The tag  has been removed.